helix-term/src/commands.rs

```diff
@@ -1200,35 +1209,35 @@ fn find_prev_char_impl(
 }
 
 fn find_till_char(cx: &mut Context) {
-    will_find_char(cx, find_next_char_impl, false, false)
+    will_find_char(cx, find_next_char_impl, false, false, Some("t".to_string()))
 }
 
 fn find_next_char(cx: &mut Context) {
-    will_find_char(cx, find_next_char_impl, true, false)
+    will_find_char(cx, find_next_char_impl, true, false, Some("f".to_string()))
 }
 
 fn extend_till_char(cx: &mut Context) {
-    will_find_char(cx, find_next_char_impl, false, true)
+    will_find_char(cx, find_next_char_impl, false, true, Some("vt".to_string()))
```

Comment:
The extend versions should not use "v". The "v" puts you in select mode but isn't considered to be a pending key in the same way that "t" is.